tool/vfx/vfxVkSection.h

```diff
@@ -309,10 +308,10 @@ class SectionResourceMapping : public Section {
   static const unsigned MemberCount = 2;
   static StrToMemberAddr m_addrTable[MemberCount];
   SubState m_state;
-  std::vector<SectionDescriptorRangeValueItem> m_staticDescriptorValue; // Contains descriptor range vuale
-  std::vector<SectionResourceMappingNode>      m_userDataNode;           // Contains user data node
+  std::vector<SectionDescriptorRangeValueItem> m_descriptorRangeValue; // Contains descriptor range vuale
```

Comment:
Perhaps fix the typo "vuale" in the comment as well?

Reply:
Done. Thanks!